content/docs/setup/kubernetes/install/cni/index.md

```diff
@@ -105,8 +148,8 @@ of many common Kubernetes environments.
     enable [network-policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy) in your cluster.
     *  Note for existing clusters this redeploys the nodes.
 
-1.  Install Istio via Helm including these options
-    `--set istio_cni.enabled=true --set istio-cni.cniBinDir=/home/kubernetes/bin`
+1.  Install Istio CNI via Helm including these options
```

Comment:
You lost the "istio_cni.enabled=true" part of the instructions.  Maybe reword to say "Install Istio CNI via Helm as described above, but also with this option."  Note also use of "these options" in original text is not proper.

Reply:
I removed the `istio_cni.enabled` since the below refers to the istio-cni param.  Maybe I will add an example it like this:

```
$ helm install istio.io/istio-cni --name=istio-cni --namespace=istio-system --set cniBinDir=/home/kubernetes/bin
```